lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -297,16 +320,20 @@ def main():
     single_transaction = module.params["single_transaction"]
     quick = module.params["quick"]
 
+    if len(db) > 1 and state != 'dump':
+        module.fail_json(msg="Multiple databases is only supported with state=%s" % state)
```

Comment:
if I pass state=absent, db=db1,db2, will I get something like:
```
Multiple databases is only supported with state=absent
```
?

Reply:
You should get something like

`Multiple databases is only supported with state=dump`
and not
`Multiple databases is only supported with state=absent`

Reference:
https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_databases

Reply:
I get your point here @Andersson007 . Fixed the code here